app/assets/javascripts/govuk_publishing_components/load-analytics.js

```diff
@@ -0,0 +1,244 @@
+//= require govuk_publishing_components/analytics
+//= require govuk_publishing_components/analytics-ga4
+
+window.GOVUK.loadAnalytics = {
+  productionDomains: [
+    'www.gov.uk',
+    'www-origin.publishing.service.gov.uk'
+  ],
+  stagingDomains: [
+    'www.staging.publishing.service.gov.uk',
+    'www-origin.staging.publishing.service.gov.uk'
+  ],
+  integrationDomains: [
+    'www.integration.publishing.service.gov.uk',
+    'www-origin.integration.publishing.service.gov.uk'
+  ],
+  developmentDomains: [
+    'localhost', '127.0.0.1', '0.0.0.0'
+  ],
+
+  // For universal analytics - cross domain tracking
+  linkedDomains: [
```

Comment:
I'm wondering whether it would make this code easier to navigate by moving `linkedDomains` into its own separate file and importing it somehow? Non-blocking, we could do this later.